This patch by Rudolf Cejka aims to fix cvsupchk for names
that contain spaces. -- Matthias Andree, 2006-11-07

--- cvsupchk.orig	Tue Oct 31 14:51:57 2006
+++ cvsupchk	Tue Oct 31 14:57:47 2006
@@ -396,6 +396,8 @@
     
     c = string.split(l)
 
+    c[1] = string.replace(c[1], "\_", " ")
+
     if c[0] not in [ 'C', 'c', 'D', 'F', 'U', 'V', 'v' ]:
       diag_fatal('Unrecognized line: "%s"' % l[0:-1])
 
